shared/people/invite-friends/invite-contacts.native.tsx

```diff
@@ -178,24 +178,30 @@ const InviteContacts = () => {
       {(!!contactsErrorMessage || !!rpcErrorMessage) && (
         <Kb.Banner color="red">{contactsErrorMessage ?? rpcErrorMessage}</Kb.Banner>
       )}
-      {loading ? (
-        <Kb.ProgressIndicator type="Huge" />
+      {successCount ? (
+        <Kb.Banner color="green">{`Success! You invited ${successCount} contacts to Keybase.`}</Kb.Banner>
```

Comment:
Looks a little weird that the list disappears. I think we could include some more content in here. Like a big green checkmark or something, and a larger close button below